src/main/java/org/apache/brooklyn/cloudfoundry/entity/VanillaCloudFoundryApplication.java

```diff
@@ -43,9 +43,10 @@
 @ImplementedBy(VanillaCloudFoundryApplicationImpl.class)
 public interface VanillaCloudFoundryApplication extends Entity, Startable, DriverDependentEntity {
 
-    @SetFromFlag("name")
-    ConfigKey<String> APPLICATION_NAME = ConfigKeys.newStringConfigKey(
-            "cloudFoundry.application.name", "Name of the application");
+    @SetFromFlag("name-app")
```

Comment:
I would like t use the `name` here. But, if I used this flag `name` this `ConfigKey` is not reached, it always return `null`, so it is not configured. I looks that a different config is loaded, I tried to find the ConfigKey which conlision with `APPLICATION_NAME` in `brooklyn-server` project, but I did not find it.
So currently does  a generic `ConfigKey`  exist which is namely `name`? would it make sense to reuse it here instead of define a new flag (such as `applicationName`)?